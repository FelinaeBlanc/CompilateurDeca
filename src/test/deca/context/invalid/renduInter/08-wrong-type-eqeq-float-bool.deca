{
    float x = 4.0;
    boolean y = true;
    if( x==y ){
        print("OK");
    } else {
        print("KO");
    }
}