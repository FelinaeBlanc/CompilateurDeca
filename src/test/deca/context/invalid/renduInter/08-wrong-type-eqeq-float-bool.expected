08-wrong-type-eqeq-float-bool.deca:4:8: Les deux expressions compar√©es ne sont pas des entiers ou des flotants